pkg/aws/testutil_test.go

```diff
@@ -62,24 +68,51 @@ const (
 	fakeRDSSnapshotName = "rds-snapshot"
 )
 
-func fakeReportItemDeleting() *clusterservice.ReportItem {
+func fakeReportItemDeleted(arn string) *clusterservice.ReportItem {
+	return &clusterservice.ReportItem{
+		ID:           arn,
+		Name:         fakeResourceIdentifier,
+		Action:       clusterservice.ActionDelete,
+		ActionStatus: clusterservice.ActionStatusComplete,
+	}
+}
+
+func fakeReportItemDeleting(arn string) *clusterservice.ReportItem {
 	return &clusterservice.ReportItem{
-		ID:           fakeRDSClientInstanceARN,
+		ID:           arn,
 		Name:         fakeRDSClientInstanceIdentifier,
 		Action:       clusterservice.ActionDelete,
 		ActionStatus: clusterservice.ActionStatusInProgress,
 	}
 }
 
-func fakeReportItemDryRun() *clusterservice.ReportItem {
+func fakeReportItemDryRun(arn string) *clusterservice.ReportItem {
 	return &clusterservice.ReportItem{
-		ID:           fakeRDSClientInstanceARN,
+		ID:           arn,
 		Name:         fakeRDSClientInstanceIdentifier,
 		Action:       clusterservice.ActionDelete,
 		ActionStatus: clusterservice.ActionStatusDryRun,
 	}
 }
 
+func fakeReportItemSkipped(arn string) *clusterservice.ReportItem {
+	return &clusterservice.ReportItem{
+		ID:           arn,
+		Name:         fakeRDSClientInstanceIdentifier,
+		Action:       clusterservice.ActionDelete,
+		ActionStatus: clusterservice.ActionStatusSkipped,
+	}
+}
+
+func fakeEc2ReportItemSkipped(arn string) *clusterservice.ReportItem {
+	return &clusterservice.ReportItem{
+		ID:           arn,
+		Name:         fakeEc2ClientInstanceIdentifier,
+		Action:       clusterservice.ActionDelete,
+		ActionStatus: clusterservice.ActionStatusSkipped,
+	}
+}
+
```

Comment:
Can we refactor these into one function something like 
```suggestion
func mockReportItemSkipped(modifyFn func(*clusterservice.ReportItem)) *clusterservice.ReportItem {
	mock := &clusterservice.ReportItem{}
	if modifyFn != nil {
		modifyFn(mock)
	}
	return mock
}
```
That way we can pass inline in the test case what kind of report item we want back. It will remove adding extra util functions like this.

Reply:
:+1: